README.md

```diff
@@ -90,17 +94,239 @@ Parties that have been disclosed to may, at their discretion, decide to pay a bo
 ### Acknowledgements
 If the reporting individual or organization would like to be recognized for their discovery, they may do so after the disclosure timeout. At the discretion of each participating team, a list of these individuals and/or organizations may recognize vulnerabilities disclosed responsibly.
 
+# bcoin addendum
```

Comment:
Maybe this should be something like: `Addendum: Disclosing vulnerabilities to the bcoin team`. Again, this feels like the most important part and so should be clear what it's describing.